src/garage/torch/algos/sac.py

```diff
@@ -260,24 +274,30 @@ def _actor_objective(self, obs, new_actions, log_pi_new_actions):
             log_pi_new_actions(torch.Tensor): Log probability of the new
                 actions on the TanhNormal distributions that they were sampled
                 from. Shape is (1, buffer_batch_size).
+            get_log_alpha_kwargs(dict): keyword args that can be used in
+                retrieving the log_alpha parameter.
 
         Returns:
             torch.Tensor: loss from the Policy/Actor.
 
         """
+        if not get_log_alpha_kwargs:
+            get_log_alpha_kwargs = {}
         with torch.no_grad():
-            alpha = self._get_log_alpha().exp()
+            alpha = self._get_log_alpha(**get_log_alpha_kwargs).exp()
         min_q_new_actions = torch.min(self.qf1(obs, new_actions),
                                       self.qf2(obs, new_actions))
         policy_objective = ((alpha * log_pi_new_actions) -
                             min_q_new_actions.flatten()).mean()
         return policy_objective
 
-    def _critic_objective(self, samples):
+    def _critic_objective(self, samples, get_log_alpha_kwargs=None):
```

Comment:
same question as before. how on earth would a caller know what kwargs to provide to a subroutine of this function?